blueflood-elasticsearch/src/main/java/com/rackspacecloud/blueflood/io/ElasticsearchRestHelper.java

```diff
@@ -464,8 +472,10 @@ public int refreshIndex(String indexName) throws IOException {
         HttpGet httpGet = new HttpGet(url);
         CloseableHttpResponse response = null;
         try {
+            this.publishPoolStats();
```

Comment:
It doesn't feel right to be updating these metrics on each call. A gauge would sample these values at a set interval, outside the normal flow of control. Is there a reason we want to update them on every call vs periodic sampling?